js-weather/js/script.js

```diff
@@ -0,0 +1,40 @@
+const xhr = new XMLHttpRequest();
+
+const WeatherMapApi = 'http://api.openweathermap.org/data/2.5/weather?q=';
+const ApiLangJa = '&units=metric&lang=ja';
+const AppIdUrl = '&appid=4b5774e9f3d2a07b84f0f2f88e486224';
+
+const todaysWeather = (response) => {
+    const obj = JSON.parse(response);
+    const weather = obj.weather[0].description;
+    const city = obj.name;
+    document.getElementById('CityName').innerHTML = "現在の" + city + "の天気は" + weather;
+}
```

Comment:
(修正不要)
```suggestion
};
```
セミコロン;をなるべく書かないJSの流派もあるようですが、ここでは他の行と同様に;を書くとよいですね